cm/util/misc.py

```diff
@@ -732,27 +732,39 @@ def set_file_metadata(conn, bucket_name, remote_filename, metadata_key, metadata
     return False
 
 
-def get_file_from_public_bucket(config, bucket_name, remote_filename, local_file):
+def get_file_from_public_location(ud, remote_filename, local_file):
```

Comment:
The first parameter should stay named `config`, given the move to a more centralized configuration management from pull request #2.